modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/BulkCopyEntriesMVCActionCommand.java

```diff
@@ -81,10 +81,34 @@ protected void doProcessAction(
 		}
 	}
 
-	private void _checkDestinationGroup(Group group) throws Exception {
-		if ((group != null) && group.isStaged() && !group.isStagingGroup()) {
+	private void _checkDestinationGroup(
+			Group group, long[] groupIds, long sourceGroupId)
+		throws Exception {
+
+		if (group.isStaged() && !group.isStagingGroup()) {
 			throw new PortalException(
-				"cannot-copy-into-the-live-version-of-a-group");
+				"cannot-copy-entries-to-the-live-version-of-a-group");
```

Comment:
The property `cannot-copy-entries-to-the-live-version-of-a-group` is missing from "Language.properties", and the following ones are no longer used:

* `cannot-copy-file-entries-to-the-live-version-of-a-group`
* `cannot-copy-file-shortcuts-to-the-live-version-of-a-group`
* `cannot-copy-folders-to-the-live-version-of-a-group`
* `cannot-copy-to-the-live-version-of-a-group`

Reply:
please see https://github.com/liferay-lima/liferay-portal/pull/4702#discussion_r1330935296